test/contrib/postgres_test.py

```diff
@@ -125,6 +135,14 @@ def test_bulk_complete(self, mock_connect):
         self.assertFalse(task.complete())
 
 
+@attr('postgres')
+class PostgresQueryWithPortTest(unittest.TestCase):
```

Comment:
Why does this need its own `TestCase`? I feel it belongs within `PostgresQueryTest` since we aren't doing any `setUp()` logic with initializing the desired DummyPostgresQuery class of choice.